draft-beck-tls-trust-anchor-ids.md

```diff
@@ -252,7 +252,7 @@ When the extension is sent in EncryptedExtensions, the `extension_data` is a Tru
 
 When the extension is sent in Certificate, the `extension_data` MUST be empty and indicates that the sender sent the certificate because the certificate matched a trust anchor identifier sent by the peer. When used in this form, the extension may only be sent in the first CertificateEntry. It MUST NOT be sent in subsequent ones.
 
-## Certificate Selection
+## Certificate Selection {#certificate-selection}
```

Comment:
Nit: You only need this if you're changing from the default label. (Default label is lowercase with hyphens.)

Reply:
```suggestion
## Certificate Selection
```